tests/__data__/feed_alt.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><rss xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:atom="http://www.w3.org/2005/Atom" version="2.0" xmlns:org="https://g.co/gci" xmlns:property="https://g.co/gci"><channel><title><![CDATA[Google Code-in Leaders]]></title><description><![CDATA[A feed for Google Code-in updates]]></description><link>https://gci-leaders.netlify.com</link><generator>RSS for Node</generator><lastBuildDate>Sun, 24 Dec 2017 03:04:35 GMT</lastBuildDate><atom:link href="https://gci-leaders.netlify.com/feed.xml" rel="self" type="application/rss+xml"/><pubDate>Sun, 24 Dec 2017 03:04:35 GMT</pubDate></channel></rss>
```

Comment:
Missing newline

Reply:
The feed generated by `rss.js` doesn't contain a trailing `\n`, and so adding it in this file would fail the test. Would it be better to add a newline to the test of leave the file as-is?